### PR TITLE
Add GraphQL Authorization Misconfiguration

### PR DESCRIPTION
**Pull request by ErebusZ**

### CODE REVIEW

File: WEB_SERVICE/WEB/_POTENTIALLY/GRAPHQL_AUTHORIZATION_MISCONFIGURATION/description.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The Provided endpoint is detected to have a potential GraphQL authentication misconfiguration where two different queries return the same object type, but one of them might lack proper authorization controls.
+This affects sensitive information that should be restricted to authorized users.
+Attackers can exploit the unprotected query to access data they shouldn't have permission to view, potentially leading to exposure of personal information, business secrets, or other confidential data.
```
Comment (elyousfi5) [inline WEB_SERVICE/WEB/_POTENTIALLY/GRAPHQL_AUTHORIZATION_MISCONFIGURATION/description.md:3]: this is the format of a technical detail, normally the description should simply explain what is GraphQL Authorization Misconfiguration (the fact of unprotected path for a query ...).

Comment (PiranhaSa) [reply WEB_SERVICE/WEB/_POTENTIALLY/GRAPHQL_AUTHORIZATION_MISCONFIGURATION/description.md:3]: +1

File: WEB_SERVICE/WEB/_POTENTIALLY/GRAPHQL_AUTHORIZATION_MISCONFIGURATION/meta.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "risk_rating": "potentially",
+  "short_description": "Broken graphQl authorization allowing attackers to query sensitive data.",
+  "references": {
+    "Exploiting a broken access control vulnerability on GraphQL (Vaadata Blog)": "https://www.vaadata.com/blog/exploiting-a-broken-access-control-vulnerability-on-graphql/",
+    "GraphQL API vulnerabilities (PortSwigger)": "https://portswigger.net/web-security/graphql"
```
Comment (PiranhaSa) [inline WEB_SERVICE/WEB/_POTENTIALLY/GRAPHQL_AUTHORIZATION_MISCONFIGURATION/meta.json:6]:
```suggestion
```

File: WEB_SERVICE/WEB/_POTENTIALLY/GRAPHQL_AUTHORIZATION_MISCONFIGURATION/description.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+A GraphQL Authorization Misconfiguration is a security vulnerability where the GraphQL API fails to properly enforce access controls on certain operations or data.
```
Comment (3asm) [inline WEB_SERVICE/WEB/_POTENTIALLY/GRAPHQL_AUTHORIZATION_MISCONFIGURATION/description.md:1]: Needs WAY more details. Needs example, vulnerable source code, fixed source code.

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**